Fix broken BR scraper.

jzohrab opened the pull request:
## Summary

Fixes, seems like it works.

Haven't tried to generalize any libs or anything, just getting the current thing to work.

lint and unit tests pass, runs locally and looks good:

`$ yarn start --location BR`: 

```
✅ Data scraped!
   - 0 cities
   - 27 states
   - 0 counties
   - 1 countries
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 143772 cases
   - 0 tested
   - 0 recovered
   - 10034 deaths
   - 0 active
```

```
data:

  {
    "country": "Brazil",
    "sources": [
      {
        "description": "Secretaria de Vigilância em Saúde do Ministério da Saúde",
        "name": "SVS-MS",
        "url": "https://covid.saude.gov.br/"
      }
    ],
    "url": "https://covid.saude.gov.br/",
    "maintainers": [
      {
        "name": "Felipe Roberto",
        "email": "contato@feliperoberto.com.br",
        "url": "http://feliperoberto.com.br",
        "github": "feliperoberto",
        "country": "iso1:BR",
        "flag": "🇧🇷"
      }
    ],
    "state": "Acre",
    "cases": 311,
    "deaths": 16,
    "population": 881935,
    "coordinates": [
      -70.3015,
      -9.13
    ],
    "aggregate": "state",
    "rating": 0.5098039215686274,
    "tz": [
      "America/Rio_Branco"
    ],
    "featureId": "iso2:BR-AC",
    "countryId": "iso1:BR",
    "stateId": "iso2:BR-AC",
    "name": "Acre, Brazil",
    "level": "state"
  },
```

matches the data on the page.

lazd commented on src/shared/scrapers/BR/index.js:
```diff
@@ -57,44 +57,111 @@ const scraper = {
+      return response;
+    },
+
+    // TODO: things actually started busting before this point .. adjust this date as we figure out where it actually works.
```
Yeah def run a timeseries and see the last date that worked:

```
yarn timeseries -d 2020-4-1
```

jzohrab replied:
Running for that location: `yarn timeseries --location BR -d 2020-4-1`, thx for tip.

jzohrab replied:
Hm, I just ran that ... and there was no failure.  So either BR just started failing today ... or something else happened, and it was never broken!  At any rate, the current scraper code will provide more failure data than before, should something go wrong.

jzohrab commented:
Interesting, testing failed, with the headings errors I added.  It works locally of course :-)

```
    Updated scraper src/shared/scrapers/BR/index.js
   Scraper failed with error: Error: Text did not match expected titles
```

I'll add more check detail, and will check the timeseries.

jzohrab commented:
This PR is good to go, IMO.

lazd commented:
LGTM!
